.github/workflows/continuous-integration-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We should put this one level up, since it's not a workflow.

In addition to that, if we agree on a pre-defined location, we can skip configuring its path.